atlasdb-dbkvs-tests/src/test/java/com/palantir/atlasdb/keyvalue/dbkvs/DbkvsPostgresKeyValueServiceTest.java

```diff
@@ -38,28 +39,18 @@ protected KeyValueService getKeyValueService() {
         return kvs;
     }
 
-    @Test
-    public void dontThrowWhenCreatingTheSameLongTable() throws Exception {
-        TableReference longTableName = TableReference.create(TEST_NAMESPACE, TEST_LONG_TABLE_NAME);
-
-        keyValueService.createTable(longTableName, AtlasDbConstants.GENERIC_TABLE_METADATA);
-        keyValueService.createTable(longTableName, AtlasDbConstants.GENERIC_TABLE_METADATA);
-
-        keyValueService.dropTable(longTableName);
-    }
-
     @Test(expected = RuntimeException.class)
     public void throwWhenCreatingDifferentLongTablesWithSameFirstCharactersUntilTheTableNameLimit() {
         String tableNameForFirstSixtyCharactersToBeSame = StringUtils.left(TEST_LONG_TABLE_NAME,
-                AtlasDbConstants.ATLASDB_POSTGRES_TABLE_NAME_LIMIT - TEST_NAMESPACE.getName().length()
+                PostgresDdlTable.ATLASDB_POSTGRES_TABLE_NAME_LIMIT - TEST_NAMESPACE.getName().length()
```

Comment:
Oracle table name mapping does slightly different modifications eg: truncating the namespace and adding a number at the end. Also, it never throws when creating long table names. So the same tests are not applicable there.